.gitlab-ci.yml

```diff
@@ -0,0 +1,29 @@
+before_script:
+  - docker info
+  # check the availability of some commands:
+  - command >/dev/null -v python3 jq
+
+build-in-docker:
+  stage: build
+  script:
+    - cd tools/docker
+    # Try to pull from dockerhub, build the container images if that fails
+    - ./image-pull.sh || ./image-build.sh
```

Comment:
For building, it should be possible to use the prplmesh-builder:ubuntu-18.04 container directly. Of course we can't use the shell runner then, so remove the shell tag.

Note that this is something we should probably do in a follow-up PR - for this PR, it's good to stay as close as possible to .travis.yml.

D'oh, I realize now that the builder container has maptools as the entrypoint, so that won't work... We'd first have to remove the entrypoint from that container so we have full flexibility for running stuff in it.